x/distribution/client/module_client.go

```diff
@@ -0,0 +1,38 @@
+package client
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	distCmds "github.com/cosmos/cosmos-sdk/x/distribution/client/cli"
+	"github.com/spf13/cobra"
+	amino "github.com/tendermint/go-amino"
+)
+
+// ModuleClient exports all client functionality from this module
+type ModuleClient struct {
+	storeKey string
+	cdc      *amino.Codec
+}
+
+func NewModuleClient(storeKey string, cdc *amino.Codec) ModuleClient {
+	return ModuleClient{storeKey, cdc}
+}
+
+// GetQueryCmd returns the cli query commands for this module
+func (mc ModuleClient) GetQueryCmd() *cobra.Command {
+	return &cobra.Command{Hidden: true}
```

Comment:
As in, no query command? maybe we can return `nil` or return a boolean to indicate whether the module has any query commands.

Reply:
Well this also works just as well.

Reply:
I don't know what `Hidden: true` does (I guess that it returns a nil-ish command that doesn't show up?)

Reply:
Keeping this in per discussion with @cwgoes